ravenframework/CodeInterfaceClasses/RELAP5/RELAPparser.py

```diff
@@ -42,10 +41,11 @@ class RELAPparser():
   """
     Import the RELAP5 input as list of lines, provide methods to add/change entries and print it back
   """
-  def __init__(self,inputFile,addMinorEdits=False):
+  def __init__(self,inputFile,datatypes=None,addMinorEdits=False):
     """
       Constructor
       @ In, inputFile, string, input file name
+      @ In, datatypes, {}, dict {floats:[],integers:[]}
```

Comment:
The default is floats, and the codes only process integers and it seems to me only integers are stored in datatypes. I would suggest update the line with integers only.

Reply:
Addressed